nella/monolog-tracy-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Nella\\MonologTracyBundle\\MonologTracyBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "log/": "%VAR_DIR%/log/"
```

Comment:
If I understand it correctly, as the `/var/` directory is gitignored in `symfony/skeleton`, it would be better to let the library automatically create the directory, instead of creating it in recipe, right?